ds3-cli-certification/src/test/java/com/spectralogic/ds3cli/certification/Certification_Test_7.java

```diff
@@ -0,0 +1,454 @@
+    private static final String NO_RIGHTS_USERNAME = "no_rights_user";
+    private static final Long GB_BYTES = 1073741824L;
+    private static CertificationWriter OUT;
+
+    private final static Ds3ExceptionHandlerMapper EXCEPTION = Ds3ExceptionHandlerMapper.getInstance();
+    static {
+        EXCEPTION.addHandler(FailedRequestException.class, new FailedRequestExceptionHandler());
+        EXCEPTION.addHandler(RuntimeException.class, new RuntimeExceptionHandler());
+    }
+
+    @BeforeClass
+    public static void startup() throws IOException {
+        LOG.setLevel(Level.INFO);
+
+        final String fileName = Certification_Test_7.class.getSimpleName() + "_" + new Date().getTime();
+        OUT = new CertificationWriter(fileName);
+
+        verifyAvailableTapePartition(client);
+    }
+
+    @AfterClass
+    public static void teardown() throws IOException {
+        client.close();
+        OUT.close();
+    }
+
+    @Test
+    public void test_7_1_create_bucket() throws Exception {
+        final String testDescription = "7.1: Create Bucket";
+        final String bucketName = CertificationUtil.getBucketName(testDescription);
+        OUT.startNewTest(testDescription);
+        boolean success = false;
+
+        try {
+            final String expectedCreateBucketMsg = "Success: created bucket " + bucketName + ".";
+            OUT.insertLog("Create bucket");
+            final String createBucketCmd = "--http -c put_bucket -b " + bucketName;
+            final CommandResponse createBucketResponse = Util.command(client, createBucketCmd);
+            OUT.insertCommand(createBucketCmd, createBucketResponse.getMessage());
+            assertThat(createBucketResponse.getMessage(), is(expectedCreateBucketMsg));
+
+            OUT.insertLog("List all buckets");
+            final String listBucketCmd = "--http -c get_service";
+            final CommandResponse listBucketResponse = Util.command(client, listBucketCmd);
+            OUT.insertCommand(listBucketCmd, listBucketResponse.getMessage());
+            assertThat(listBucketResponse.getMessage(), containsString(bucketName));
+            success = true;
+        } finally {
+            OUT.finishTest(testDescription, success);
+            Util.deleteBucket(client, bucketName);
+        }
+    }
+
+    @Test
+    public void test_7_2_1_invalid_credentials() throws Exception {
+        final String testDescription = "7.2.1: Invalid Credentials";
+        final String endpoint = System.getenv("DS3_ENDPOINT");
+        final Credentials creds = new Credentials("invalid_access_id", "invalid_secret_key");
+        final Ds3Client invalid_client = Ds3ClientBuilder.create(endpoint, creds).withHttps(false).build();
+        OUT.startNewTest(testDescription);
+        boolean success = false;
+
+        try {
+            Util.command(invalid_client, "--http -c get_service");
```

Comment:
This should have a fail() statement after the Util.command, otherwise you'll get a false positive